aspnetcore/signalr/get-started-signalr-core/sample/chat.js

```diff
@@ -0,0 +1,25 @@
+﻿var connection = new signalR.HubConnection('chathub');
+connection.start().catch(err => showErr(err));
+
+connection.on('Send', (timestamp, user, message) => {
```

Comment:
Put the call to `.on` before the call to `.start`. Otherwise you could technically miss invocations that occur immediately after the connection is started but before you've bound the handler. That's the main reason we have a `.start` method rather than just starting the connection in the constructor ;)